pkgs/by-name/ce/cegui/package.nix

```diff
@@ -37,8 +36,6 @@ stdenv.mkDerivation {
       expat
     ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
-      darwin.apple_sdk.frameworks.Cocoa
-      darwin.apple_sdk.frameworks.Foundation
       libiconv
     ];
```

Comment:
This can also be removed; `libiconv` is included in the Darwin `stdenv` these days. (It means something on OpenBSD, I think, but there’s at least no reason to keep `lib.optionals stdenv.hostPlatform.isDarwin [ libiconv ]` as‐is.)